server/public/index.html

```diff
@@ -5,6 +5,12 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>sessionLite</title>
   <link rel="icon" href="/favicon.ico">
+  <meta name="og:title" content="sessionLite">
+  <meta name="og:type" content="article" />
+  <meta name="og:image"
+    content="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQd6Mp-TWV2A_phMcrxhCIS7C-D-g1LpXQA9Q&usqp=CAU">
```

Comment:
You should download this image and put it in your `server/public/` directory so that you aren't relying on someone else hosting it.

```html
<meta name="og:image"
    content="https://session-lite.herokuapp.com/open-graph-image.jpeg">
```